scenarios/hypertension/hypertension_contactable_overdue_patients.feature

```diff
@@ -0,0 +1,262 @@
+      | 6_MonthsAgo  | 0 |
+      | 7_MonthsAgo  | 0 |
+      | 8_MonthsAgo  | 0 |
+      | 9_MonthsAgo  | 0 |
+      | 10_MonthsAgo | 0 |
+      | 11_MonthsAgo | 0 |
+      | 12_MonthsAgo | 0 |
+
+  Scenario: Dead patients should not be included
+    Given I am signed in as a user with role "Superuser"
+    And I have access to an organisation unit at level 5
+    And I register that organisation unit for program "Hypertension & Diabetes"
+
+    And I create a new TEI on "7_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Test         |
+      | GEN - Family name                     | TEST         |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | YES          |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | Rose Gardens |
+      | Patient Phone Number                  | 345672781624 |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Hypertension & Diabetes visit" event scheduled for "6_MonthsAgo"
+
+    And I create a new TEI on "7_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Test         |
+      | GEN - Family name                     | TEST         |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | YES          |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | Rose Gardens |
+      | Patient Phone Number                  | 345672781624 |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Hypertension & Diabetes visit" event scheduled for "6_MonthsAgo"
+    And That TEI was updated on "2_MonthsAgo" with the following attributes
+      | HTN - NCD Patient Status | DIED |
+
+    When I wait for 1 second
+    And I export the analytics
+
+    Then The value of "PI":"HTN - Contactable overdue patients" with period type "Months" should be
+      | thisMonth    | 1 |
+      | 1_MonthAgo   | 1 |
+      | 2_MonthsAgo  | 1 |
+      | 3_MonthsAgo  | 1 |
+      | 4_MonthsAgo  | 1 |
+      | 5_MonthsAgo  | 1 |
```

Comment:
Should the count be 2 for periods prior to 1_MonthAgo and then 1 for the rest of them ?